packages/manager/modules/pci/src/projects/project/notebooks/add/components/notebook-attach/notebook-attach.html

```diff
@@ -8,55 +7,93 @@
                                         >
-                                            <option data-ng-bind="'pci_notebook_add_attach_container_storage_container_select' | translate" default></option>
-                                            <option
-                                                ng-repeat="storage in $ctrl.storages track by storage.id"
-                                                data-ng-bind="::storage.name"
-                                                value="{{storage.name}}"
-                                            >
-                                            </option>
-                                        </select>
-                                        <span
-                                            class="oui-icon oui-icon-chevron-down"
-                                            aria-hidden="true"
-                                        ></span>
+                                        </oui-select>
                                     </label>
                                 </oui-field>
                             </div>
+                            <div
+                                class="oui-inline-adder__field"
+                                ng-if="volume.privateSwift"
+                            >
+                                <oui-field
+                                    data-label="{{:: 'pci_projects_project_training_jobs_submit_data_prefix' | translate  }}"
+                                    data-label-popover="{{:: 'pci_projects_project_training_jobs_submit_data_prefix_help' | translate }}"
+                                >
+                                    <input
+                                        type="text"
+                                        class="oui-input"
+                                        placeholder=""
+                                        data-ng-init="volumePrefixId = 'volumePrefix-' + $index"
+                                        data-ng-attr-name="{{volumePrefixId}}"
+                                        data-ng-attr-id="{{volumePrefixId}}"
+                                        data-ng-model="volume.prefix"
+                                        data-ng-pattern="$ctrl.NOTEBOOK_ATTACH_STORAGE.PREFIX_REGEX"
+                                    />
+                                </oui-field>
+                            </div>
+                            <div
+                                class="oui-inline-adder__field"
+                                ng-if="volume.publicGit"
+                            >
+                                <oui-field
+                                    label="{{:: 'pci_notebook_add_attach_container_git_url' | translate}}"
+                                >
+                                    <input
+                                        type="text"
+                                        class="oui-input"
+                                        placeholder=""
```

Comment:
If it's not needed then you can remove the `placeholder` attribute. Don't know if it's the expected behavior.

```suggestion
```

Reply:
removed